src/main/resources/assets/js/app/visualization/constants.ts

```diff
@@ -34,7 +33,7 @@ const COLORS = {
     ]
 };
 
-const RENDER_CONFIG: RenderConfig = {
+const RENDER_CONFIG = () => ({
```

Comment:
It's not a good practice to upper-case function names (as it's no longer a constant). Rename this method to `getRenderConfig`